CSP Test Fail

Hi Guys, The CSP test is failing on my machine. I am using the dev version of mne-python, and have not modified anything for this test. I am running OS X with anaconda with mkl. Any suggestions/ideas as to why this is happening? Below is the message I get.
Thanks!
======================================================================
FAIL: Test Common Spatial Patterns algorithm on epochs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lamexicana/anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/lamexicana/repos/mne-python/mne/decoding/tests/test_csp.py", line 76, in test_csp
    assert_array_almost_equal(csp.filters_, csp_epochs.filters_, -1)
  File "/Users/lamexicana/anaconda/lib/python2.7/site-packages/numpy/testing/utils.py", line 842, in assert_array_almost_equal
    precision=decimal)
  File "/Users/lamexicana/anaconda/lib/python2.7/site-packages/numpy/testing/utils.py", line 665, in assert_array_compare
    raise AssertionError(msg)
AssertionError: 
Arrays are not almost equal to -1 decimals

(mismatch 11.1111111111%)
 x: [repr failed]
 y: [repr failed]
-------------------- >> begin captured stdout << ---------------------
Opening raw data file /Users/lamexicana/repos/mne-python/mne/io/tests/data/test_raw.fif...
    Read a total of 3 projection items:
        PCA-v1 (1 x 102)  idle
        PCA-v2 (1 x 102)  idle
        PCA-v3 (1 x 102)  idle
Current compensation grade : 0
    Range : 25800 ... 40199 =     42.956 ...    66.930 secs
Ready.
Adding average EEG reference projection.
1 projection items deactivated
15 matching events found
Created an SSP operator (subspace dimension = 3)
4 projection items activated
Reading 2425 ... 2845  =      4.038 ...     4.737 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 2851 ... 3271  =      4.747 ...     5.446 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 4105 ... 4525  =      6.835 ...     7.534 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 4530 ... 4950  =      7.542 ...     8.242 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 5745 ... 6165  =      9.565 ...    10.264 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 6181 ... 6601  =     10.291 ...    10.990 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 7405 ... 7825  =     12.329 ...    13.028 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 7792 ... 8212  =     12.973 ...    13.673 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 9036 ... 9456  =     15.045 ...    15.744 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 9508 ... 9928  =     15.830 ...    16.530 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 10656 ... 11076  =     17.742 ...    18.441 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 11087 ... 11507  =     18.459 ...    19.159 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 12406 ... 12826  =     20.655 ...    21.355 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 12791 ... 13211  =     21.297 ...    21.996 secs...
[done]
Applying baseline correction ... (mode: mean)
Reading 14006 ... 14399  =     23.319 ...    23.974 secs...
[done]
Applying baseline correction ... (mode: mean)
1 bad epochs dropped

--------------------- >> end captured stdout << ----------------------
>>  raise AssertionError('\nArrays are not almost equal to -1 decimals\n\n(mismatch 11.1111111111%)\n x: [repr failed]\n y: [repr failed]')
    

----------------------------------------------------------------------
Ran 50 tests in 49.692s

FAILED (SKIP=4, failures=1)

--- a/mne/decoding/tests/test_csp.py
+++ b/mne/decoding/tests/test_csp.py
@@ -31,9 +31,9 @@ def test_csp():
     events = read_events(event_name)
     picks = pick_types(raw.info, meg=True, stim=False, ecg=False,
                        eog=False, exclude='bads')
-    picks = picks[2:9:3]
+    picks = picks[2:9:3]  # subselect channels -> disable proj!
     epochs = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
-                    baseline=(None, 0), preload=True)
+                    baseline=(None, 0), preload=True, proj=False)
     epochs_data = epochs.get_data()
     n_channels = epochs_data.shape[1]
 
@@ -76,7 +76,7 @@ def test_csp():
     for attr in ('filters_', 'patterns_'):
         corr = np.corrcoef(getattr(csp, attr).ravel(),
                            getattr(csp_epochs, attr).ravel())[0, 1]
-        assert_true(corr > 0.95, msg='%s < 0.95' % corr)
+        assert_true(corr >= 0.95, msg='%s < 0.95' % corr)
 
     # make sure error is raised for undefined estimation method
     csp_fail = CSP(cov_est="undefined")

--- a/mne/decoding/tests/test_csp.py
+++ b/mne/decoding/tests/test_csp.py
@@ -74,9 +74,9 @@ def test_csp():
     csp_epochs = CSP(cov_est="epoch")
     csp_epochs.fit(epochs_data, y)
     for attr in ('filters_', 'patterns_'):
-        assert_true(np.corrcoef(
-            getattr(csp, attr).ravel(),
-            getattr(csp_epochs, attr).ravel())[0, 1] > 0.95)
+        corr = np.corrcoef(getattr(csp, attr).ravel(),
+                           getattr(csp_epochs, attr).ravel())[0, 1]
+        assert_true(corr > 0.95, msg='%s < 0.95' % corr)
 
     # make sure error is raised for undefined estimation method
     csp_fail = CSP(cov_est="undefined")

--- a/mne/decoding/tests/test_csp.py
+++ b/mne/decoding/tests/test_csp.py
@@ -73,8 +73,10 @@ def test_csp():
     # test covariance estimation methods (results should be roughly equal)
     csp_epochs = CSP(cov_est="epoch")
     csp_epochs.fit(epochs_data, y)
-    assert_array_almost_equal(csp.filters_, csp_epochs.filters_, -1)
-    assert_array_almost_equal(csp.patterns_, csp_epochs.patterns_, -1)
+    for attr in ('filters_', 'patterns_'):
+        assert_true(np.corrcoef(
+            getattr(csp, attr).ravel(),
+            getattr(csp_epochs, attr).ravel())[0, 1] > 0.95)
 
     # make sure error is raised for undefined estimation method
     csp_fail = CSP(cov_est="undefined")

